website/playground/static/index.html

```diff
@@ -128,12 +129,18 @@
   search: document.getElementById('search'),
 }, {lineNumbers: true});
 
-codeEditor.on('change', function() {
+codeEditor.setValue(atob(location.hash.slice(1)));
```

Comment:
should we keep the `if (location.hash) { ... }` around this?

Reply:
Sure. Done.